docs/home/changelog/hotfixes-2-13.md

```diff
@@ -32,12 +54,43 @@ The following is a list of the known and fixed issues for Run:ai V2.13.
+* <!--RUN-12349	Enable quota management by default -->Quota management is now enabled by default.
 
 | Internal ID | Description  |
 | ---------------------------- | ---- |
 | RUN-12923 | Fixed an issue in upgrading due to a misconfigured image for airgapped systems in 2.13.19. The helm chart contained an error, and the image is not used even though it is packaged as part of the tar. |
 | RUN-12928, </br>RUN-12968 | Fixed an issue in upgrading Prometheus due to a misconfigured image for airgapped systems in 2.13.19. The helm chart contained an error, and the image is not used even though it is packaged as part of the tar. |
+| RUN-12751 | Fixed an issue when upgrading from 2.9 to 2.13 results with a missing engine-config file. |
+| RUN-12717 | Fixed an issue where the user that is logged in as researcher manager can't see the clusters. |
+| RUN-12642 | Fixed an issue where Assets-sync could not restart due to failing to get token from control plane. |
+| RUN-12191 | Fixed an issue where there was a Time out while waiting for the `runai_allocated_gpu_count_per_project` metric to return values. |
+| RUN-10474 | Fixed an issue where the `runai-conatiner-toolkit-exporter` DaemonSet fails to start. |
+
+## Version 2.13.19 - September 27, 2023
+
+### Release content
+
+* <!--RUN-12508  Identify and present Kubeflow Notebooks in Jobs table -->Added the ability to identify Kubeflow notebooks and display them in the Jobs table.
+* <!-- RUN-12507 Support scheduling of Kubeflow workloads -->Added the ability to schedule Kubelow workloads.
+* <!--RUN-12420 Show only logged-in user's jobs -->Added functionality that displays Jobs that only belong to the user that is logged in.
+* <!-- RUN-12405 Refine out of the box alerts to run:ai rules --> Added and refined alerts to the state of Run:ai components, schedule latency, and warnings for out of memory on Jobs.
+* <!-- RUN-11711 Pod Security Admission issues in control plane -->Added the ability to work with restricted PSA policy.
+
+### Fixed issues
+
+| Internal ID | Description  |
+|-------------- | ------------------- |
+| RUN-12650 | Fixed an issue that used an incorrect metric in analytics GPU ALLOCATION PER NODE panel. Now the correct allocation is in percentage. |
+| RUN-12602 | Fixed an issue in `runaiconfig` where `WorkloadServices` have any memory requests/limits and cpu requests/limits and is overwritten with a default. |
+| RUN-12587 | Fixed an issue where the scheduler enters an endless loop running elastic jobs. |
+| RUN-12585 | Fixed an issue where the workload-controller creates a delay in running jobs. |
+| RUN-12031 | Fixed an issue when upgrading from 2.9 to 2.13 where the Scheduler pod fails to upgrade due to the change of owner. |
+| RUN-11091  | Fixed an issue when `departments FF` is off you are not able to schedule non-preemable jobs. |
```

Comment:
department FF is not customer facing. Use th following customer facing option:
"Enable Departments" option under Settings

Reply:
@omerbenedictrunai
```suggestion
| RUN-11091  | Fixed an issue where the *Departments* feature is disabled, you are not able to schedule non-preemable jobs. |
```